test/page-objects/searchItemPage.js

```diff
@@ -0,0 +1,24 @@
+import { Label } from "../../framework/elements/index.js"
+import BasePage from "../../framework/page/BasePage.js"
+
+class searchItemPage extends BasePage {
```

Comment:
class name should start from capital letter

Reply:
Fixed